src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -339,11 +366,14 @@ export const changeBrowserLocation: ActionCreator<ThunkAction<
       focusInstruction.focus = activeEnsObjectId;
     }
 
-    browserMessagingService.send('bpane', {
-      stick: `${locationData.genomeId}:${chrCode}`,
-      goto: `${startBp}-${endBp}`,
-      'message-counter': messageCount,
-      ...focusInstruction
+    browserMessagingService.send({
+      action: ChromeToBrowserMessagingActions.SET_FOCUS_LOCATION,
+      payload: {
+        stick: `${locationData.genomeId}:${chrCode}`,
+        goto: `${startBp}-${endBp}`,
+        'message-counter': messageCount,
```

Comment:
no need for message counter after https://github.com/Ensembl/ensembl-client/pull/333